oak/server/rust/oak_runtime/src/runtime/mod.rs

```diff
@@ -594,6 +627,12 @@ impl Runtime {
             write_handle,
             read_handle,
         );
+
+        self.introspection_event(EventDetails::ChannelCreated(ChannelCreated {
```

Comment:
Is this event really needed? I noticed the `ChannelDestroyed` event is never used, and it does not seem very straightforward when that one should be created. 

In general, I think it is nice to keep these events in sync with changes to the `node_infos`. Looks like there is no specific change to the `node_infos` that corresponds to the creation of a channel. So perhaps this could be removed. Especially since it is covered by the `HandleCreated` event, which has a matching `HandleDestroyed` event.

Reply:
Fair point. :) I does seem like the `Handle` events should give us sufficient information where this event could technically be omitted. 

The goal currently is to still implement a `ChannelDestroyed` event down the road, but the implementation is currently still unclear, since it requires refactoring the `introspection_event ` method. See details here: https://github.com/project-oak/oak/issues/913#issuecomment-661794936

In principle I do think that events for channels are still useful. They allow for easier updating, since channels are represented in the introspection tools. You are correct though that their creation & destruction could be deducted from `Handle` (and the yet to be added `Message`) events.